third-party/crowdsec/docker-compose.yml

```diff
@@ -9,7 +9,9 @@ services:
     command:
       - --generate-keystore
     volumes:
-      - ./sample-configuration/logback.xml:/root/conf/logback.xml
+#      - ./sample-configuration/logback.xml:/root/conf/logback.xml
```

Comment:
Why comment this?

Reply:
Because it is easier for me to check log. I will uncomment it later